submissions/Oleksii_Drohachov/Frogger_task/app.js

```diff
@@ -0,0 +1,182 @@
+
+const tileWidth = 100;
+const tileHeight = 83;
+const numberOfRows = 6;
+const numberOfColumns = 5;
+const fieldWidth = numberOfColumns * tileWidth;
+const fieldHeight = numberOfRows * tileHeight;
+const fieldBottomGap = 100;
+const emenyRowsModifier = 0.66;
```

Comment:
0.66 looks weird. Do you research as to sprite images characteristics. Do not rely on some else's calculations/numbers, they may be wrong.